AcademyApp/Model/MockData.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  MockData.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 28/06/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+enum MockData {
```

Comment:
some comment about temp mock data before networking layer prepared

Reply:
Added comment to MockData.